### ISSUE TITLE
There is no release

### ISSUE THREAD

**Issue by TheSeanLavery** [redbluegames/unity-notnullattribute#13]
Would be really nice to have a UnityPackage installer for this to easily add this to a project.

**Comment by edwardrowe** [comment]
Yeah I'm a big fan of the packages. This is a good suggestion, but I'm now recommending that people use Odin Inspector instead. Hopefully that's an option for you. It's well worth the money.

I'm gonna leave this open, as it's still relevant. But it's unlikely we will do anything on this ourselves.
